docs/statistics.rst

```diff
@@ -28,6 +28,11 @@ Statistics
 
 .. automethod:: onecodex.models.collection.SampleCollection.beta_diversity
 
+``beta_diversity_stats``
+==================
```

Comment:
I think RST requires the `====` line to match the number of characters in the line above it